src/commands/handlers/messaging.js

```diff
@@ -70,8 +70,8 @@ var handlers = {
 
         if ((message.charAt(0) === '\01') && (message.charAt(message.length - 1) === '\01')) {
             // CTCP request
-            if (message.substr(1, 6) === 'ACTION') {
-
+            var ctcp_command = message.slice(1, -1).split(' ')[0];
```

Comment:
Why not tack on `.toUpperCase()` here and avoid running the string operation for every `if`?